mkosi.conf.d/20-kali/mkosi.conf

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=kali
+
+[Distribution]
+Repositories=non-free-firmware
+
+[Host]
+ToolsTreeDistribution=kali
```

Comment:
Please drop this. To build a Kali tools tree, the host system has to be Kali as well, otherwise the keyring won't be available since it's not shipped anywhere else (yet). But if the host system is Kali, the tools tree distribution is automatically chosen to be Kali already, so setting this is pointless.

Reply:
Because this was added just to fulfill your requirement of being able to run `mkosi -d kali --tools-tree -t disk -f qemu` from the mkosi repository, which I don't need for my use-case, I'm fine with breaking this command for people who installed the keyring manually on non-Kali setups.

A new version was pushed with this statement removed.